bpfman-operator/controllers/bpfman-agent/common.go

```diff
@@ -533,6 +589,7 @@ func ProcessMapOwnerParam(ctx context.Context,
 	}
 }
 
+// get Clientset returns a kubernetes clientset.
 func getClientset() (*kubernetes.Clientset, error) {
 	kubeconfig := os.Getenv("KUBECONFIG")
```

Comment:
Yeah this still isn't the right way to do this when creating a clientset inside a k8s cluster, see my other comment!